pkg/operation/botanist/component/resourcemanager/resource_manager.go

```diff
@@ -562,6 +562,8 @@ func (r *resourceManager) ensureConfigMap(ctx context.Context, configMap *corev1
 			Enabled: true,
 			NamespaceSelectors: []metav1.LabelSelector{
 				{MatchLabels: map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleShoot}},
+				{MatchLabels: map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleIstioIngress}},
+				{MatchLabels: map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleIstioSystem}},
```

Comment:
Don't we need https://github.com/gardener/gardener/blob/278a36cd978dce8f3ea1c3f2a946c9afbd55b93e/pkg/gardenlet/controller/networkpolicy/reconciler.go#L210 as well? cc @axel7born

Reply:
See https://github.com/gardener/gardener/pull/7570#discussion_r1120169974.